README.md

```diff
@@ -44,6 +44,18 @@ twigcs /path/to/views --exclude vendor
 
 Tips: You can use multiple _exclude_ parameters.
 
+## Restricting output
+
+By default TwigCS will output all lines that have violations regardless of whether they match the severity level
+specified or not. If you only want to see errors that are greater than or equal to the severity level you've specified
+you can use the `--display` option. For example. 
+
+```bash
+twigcs /path/to/views --severity error --display blocking
```

Comment:
Can you add the `all` option in documentation to present all possibilities at this option ?